### PR TITLE
docs(pagination): Improve documentation for Pagination component page

### PR DESCRIPTION
**Pull request by willamesoares**
Adjust sentences and structure for description block.

### CODE REVIEW

File: docs/css/pagination.md [hunk 1]
```diff
@@ -8,17 +8,16 @@ path: pagination
...
 
-The pagination component should be wrapped in a `nav` element to identify it as a
-navigation section to screen readers and other technologies. You can use `.align-center` or `.align-right` in order to align pagination items.
-
-Links are customizable for different circumstances. Use `.disabled` for unclickable links and `.active` to indicate the current page.
+The component should be wrapped in a `nav` element to identify it as a
+navigation section.  
 
-The `.pagination-arrow` class can be used to modify the background and color for the next and previous arrows on mouse hover.
+To customize links according to their state you can use the class `.disabled` for unclickable links and `.active` to indicate the current page. To modify the background and color for the next and previous arrows on mouse hover, you can use the `.pagination-arrow` class, as shown below.
```
Comment (felipesoares6) [inline docs/css/pagination.md]: this `color` where is that from?

Comment (willamesoares) [inline docs/css/pagination.md]: what do you mean?

Comment (felipesoares6) [inline docs/css/pagination.md]: This `color` it's from background ? I think you could  be more specific using `background color`

Comment (willamesoares) [inline docs/css/pagination.md]: The problem is that this class modifies both the background color and the color of the element itself.

**Pull request merged**